cms/templates/js/add-xblock-component-button.underscore

```diff
@@ -1,8 +1,8 @@
 <% if (type === 'advanced' || templates.length > 1) { %>
-<a href="#" class="multiple-templates add-xblock-component-button" data-type="<%= type %>">
+<button type="button" class="multiple-templates add-xblock-component-button" data-type="<%= type %>">
```

Comment:
Would you add (in addition to the others that are here) a class on this button `button-add-component`?